submissions/lawalLolade/html-forms/contactFormPage.html

```diff
@@ -0,0 +1,79 @@
+<html lang="en">
+<head>
```

Comment:
Also, I can see that you already have contactFormPage before, Why are you creating a new one?
![image](https://user-images.githubusercontent.com/43061141/101980807-6729f380-3c68-11eb-9c4c-e1f72fcd4a98.png)